app/ts/util/significantDigits.ts

```diff
@@ -1,6 +1,6 @@
 export default function significantDigits(x: number, precision?: number) {
   if (x === undefined || x === null || isNaN(x)) {
-    throw 'attempt to apply significant digits to non-numeric value';
+    throw `attempt to apply significant digits to non-numeric value ${x}`;
```

Comment:
is this tested?

Reply:
apparently not, since it broke nothing. It appears that `toThrow` uses `startsWith` to check for message equality :S